platform/ios/src/MGLMapView.h

```diff
@@ -856,6 +856,15 @@ MGL_EXPORT IB_DESIGNABLE
  */
 - (void)setCamera:(MGLMapCamera *)camera animated:(BOOL)animated;
 
+/**
+ Moves the viewpoint to a different location without using a transition.
+ 
+ @param camera The new viewpoint.
+ @param edgePadding The minimum padding (in screen points) that would be visible
+ 
+ */
+- (void)setCamera:(MGLMapCamera *)camera edgePadding:(UIEdgeInsets)edgePadding;
```

Comment:
This method should accept an `animated` parameter. Just as there’s a `-setVisibleCoordinateBounds:animated:` and `-setVisibleCoordinateBounds:edgePadding:animated:`, there should be `-setCamera:animated:` and `-setCamera:edgePadding:animated:`. It seems unfair to force the developer to choose between specifying an edge padding and using the built-in animation capabilities, even if your particular use case doesn’t require those capabilities.